include/fluent-bit/flb_config.h

```diff
@@ -34,6 +34,9 @@
 #define FLB_CONFIG_FLUSH_SECS   5
 #define FLB_CONFIG_HTTP_LISTEN  "0.0.0.0"
 #define FLB_CONFIG_HTTP_PORT    "2020"
+#define ERROR_COUNTS_DEFAULT 5
+#define RETRY_FAILURE_COUNTS_DEFAULT 5
+#define PERIOD_BY_SECOND 60
```

Comment:
Here and in the structure, this variable name is odd IMO. "period by seconds" isn't right, you'd say "period in seconds".

Also, here and in the struct, this variable name is not descriptive enough- this is the `flb_config` file/struct. It's not clear that this is used in the health check.

Something like `HEALTH_CHECK_PERIOD` would be ideal.